openapi/openapi/src/main/scala/endpoints4s/openapi/model/OpenApi.scala

```diff
@@ -1,21 +1,22 @@
 package endpoints4s.openapi.model
 
-import java.io.Serializable
-
-import endpoints4s.{Encoder, Hashing}
 import endpoints4s.algebra.{ExternalDocumentationObject, Tag}
+import endpoints4s.{Encoder, Hashing}
 
-import scala.collection.mutable
+import java.io.Serializable
+import scala.collection.{Map, mutable, immutable}
 
 /** @see [[https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md]]
   * @note Throws an exception on creation if several tags have the same name but not the same other attributes.
   */
 final class OpenApi private (
     val info: Info,
-    val paths: Map[String, PathItem],
+    private val innerPaths: Map[String, PathItem],
```

Comment:
I think it should be possible to use `scala.collection.SeqMap` instead, to make sure we don’t transform it into a `Map` type that is not ordered, by mistake.

After a second read, I realize that this may be difficult because the method `OpenApi.apply` would have an incompatible signature (maybe this would be fixed by adding an overload, though).

Reply:
Will check, thanks for the quick feedback!